src/searchPage/stillingsøk/StillingSøkeboks.less

```diff
@@ -0,0 +1,47 @@
+@import '../../variables';
+
+.Søkeboks {
```

Comment:
Vi bruker liten forbokstav og kebab-case for klassenavn i ny kode. Burde vel vært dokumentert et sted 😅

Reply:
Aha! Den er grei!

Reply:
Skal fila i seg selv også ha liten forbokstav?